src/containers/Shared.tsx

```diff
@@ -201,11 +198,6 @@ const Shared = ({ children, ...rest }: Props) => {
 
     async function accountChecker() {
         if (!account || !chainId || !provider || !geb) return
-        popupsActions.setWaitingPayload({
```

Comment:
Why did we remove the initializing step? I think that was helpful for easing these transitions

Reply:
@pi0neerpat I've added back the initializing modal and the timeouts. The reason I added back the timeouts is when I was on a throttled network I was getting network errors that went away when I removed the timeouts, so it's something to troubleshoot.

The stepper/onboarding section still does not persist when the initializing modal is open and I've already spent a while trying to figure out why this is to no success. Even when I totally remove the modal styling (see video below) the background disappears. I've also tried removing all the setBlockBackdrop(true) calls, removing all the contents in Shared.tsx that might be related (like BlockBodyContainer, WaitingModal), checking for conditional rendering of the Stepper component, removing the setBlockBody call...

So if we want to keep the initializing modal then we should keep this behavior for now but somehow there's some conditional rendering somewhere preventing the onboarding/stepper section from rendering immediately when the initializing modal is open, or it could be the case that it just takes a while to render these both at the same time when refreshing the page or moving to the next step

Backdrop disappears even when WaitingModal is gone

https://github.com/open-dollar/od-app/assets/47253537/d300c587-9e3b-4a07-aed5-fd1a9a3fca2c